WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -688,18 +688,21 @@ private async Task<Transaction> ProceedWithSigningStateAsync(uint256 roundId, Im
 		var signingState = roundState.Assert<SigningState>();
 		var unsignedCoinJoin = signingState.CreateUnsignedTransaction();
 
-		// Sanity check.
-		if (!SanityCheck(outputTxOuts, unsignedCoinJoin))
-		{
-			string round = roundState.BlameOf == 0 ? "Round" : "Blame Round";
-
-			throw new InvalidOperationException($"{round} ({roundState.Id}): My output is missing.");
-		}
+		// If everything is okay the sign all the inputs. Otherwise, in case there are missing outputs, the server is
+		// lying (it lied us before when it responded with 200 OK to the OutputRegistration requests or it is lying us
+		// now when we identify as satoshi.
+		// In this scenario we should ban the coordinator and stop dealing with it.
```

Comment:
Not sure if this is needed in the comments, other than a future todo. 
```suggestion
```

Reply:
What it is not in the code it is hidden from the devs.

Reply:
No strong opinion on this, but the issue reference would be enough IMO. Long comments make fragmented code.